src/test/regress/expected/turn_mx_on_0.out

```diff
@@ -0,0 +1,16 @@
+ALTER SYSTEM SET citus.enable_metadata_sync_by_default TO ON;
```

Comment:
I want to get rid of these as quickly as possible :)

Reply:
Me too